shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -111,8 +111,11 @@
 // |shell::PlatformView|
 void PlatformViewIOS::UpdateSemantics(blink::SemanticsNodeUpdates update,
                                       blink::CustomAccessibilityActionUpdates actions) {
+  FML_DCHECK(owner_controller_);
   if (accessibility_bridge_) {
     accessibility_bridge_->UpdateSemantics(std::move(update), std::move(actions));
+    [[NSNotificationCenter defaultCenter] postNotificationName:@"FlutterSemanticsUpdate"
```

Comment:
Should use `FlutterSemanticsUpdateNotification`. =)

Reply:
Done.  I had to switch what header it was in to make sure I get one that the right people have access to.  It makes more sense in FutterViewController.h anyway since that's the object that'll get passed through.  We may have to rethink if we want semantics to be enabled for an engine that doesn't have a ViewController though